submissions/samuelJegede/database/transportation database.sql

```diff
@@ -0,0 +1,28 @@
+CREATE DATABASE transport;
+USE transport;
+CREATE TABLE passengers (
+	id INT NOT NULL AUTO_INCREMENT,
+    full_name VARCHAR(250) NOT NULL,
+    sex VARCHAR(6) NOT NULL,
+    age VARCHAR(3),
```

Comment:
Integer is fine for age. So if someone types 0I5 it can be flagged as wrong instead of 015